src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -89,6 +89,7 @@ def create_llm_inputs(
         add_stream: bool = False,
```

Comment:
At a high level, what's the purpose of concatenating the prompts into one string?

Reply:
The API requires either a string or list of strings.
Previously, we took all inputs and concatenated them into a single string as a means to provide a more streamlined experience. This enables the other half of the behavior but requires a new flag to control the behavior